packages/mdadm/mdadm-tmpfiles.conf

```diff
@@ -0,0 +1,2 @@
+d /var/run/mdadm 0700 chrony chrony -
```

Comment:
This should be an mdadm user account (or whatever the standard is). You can create that via a sysusers.conf drop-in like in the chrony package.